plugins/dynatrace/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@backstage/plugin-dynatrace",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/catalog-model": "^1.0.2",
+    "@backstage/core-components": "^0.9.3-next.0",
+    "@backstage/core-plugin-api": "^1.0.0",
+    "@backstage/theme": "^0.2.15",
+    "@material-ui/core": "^4.9.13",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "4.0.0-alpha.57",
+    "cross-fetch": "^3.1.5",
+    "react-use": "^17.2.4"
+  },
+  "peerDependencies": {
+    "@backstage/plugin-catalog-react": "^1.0.0",
+    "react": "^16.13.1 || ^17.0.0"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.16.1-next.0",
+    "@backstage/core-app-api": "^1.0.1-next.0",
```

Comment:
There has been a backstage release since this PR was created so please make sure that the versions for the `@backstage/` packages correspond with the versions currently in master, that's likely why CI is failing currently :)

Reply:
will do!